<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConsoleDialog</class>
 <widget class="QDialog" name="ConsoleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Console</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>10</y>
     <width>591</width>
     <height>421</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
